codebase/config/sync/core.entity_form_display.node.collection_object.default.yml

```diff
@@ -121,16 +121,6 @@ content:
     third_party_settings: {  }
     type: entity_reference_autocomplete
     region: content
-  field_model:
```

Comment:
Do you know what the consequence of removing the model field from the collection object is?  Is this intended to be a part of the PR?

Reply:
This is remove the field from the edit form for a collection object. This doesn't remove the field form the content type. The result is that users will not see this field. They should never edit this field as it's set to "Collection" and that's all this can be. 

However, this doesn't need to be in this PR and I'll take this out.

Reply:
What's odd is that this file here shows this but if you look at the config_changes branch at this file, this change is not there.

Reply:
Git is doing something weird here and this code is outdated. I'm going to resolve this issue.